Library/src/main/java/com/infomaniak/lib/login/LoginWebViewClient.kt

```diff
@@ -87,7 +87,7 @@ open class LoginWebViewClient(
     }
 
     override fun onReceivedHttpError(view: WebView?, request: WebResourceRequest?, errorResponse: WebResourceResponse?) {
-        if (isValidUrl(request?.url.toString())) errorResult(HTTP_ERROR_CODE)
+        if (!isValidUrl(request?.url.toString())) errorResult(HTTP_ERROR_CODE)
```

Comment:
We should probably revert the previous version instead, or even better, test the 2 bugs at once to find the ultimate solution.